<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyInfo</class>
 <widget class="QWidget" name="modifyInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="SearchBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>0</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; text-decoration: underline;&quot;&gt;骑手信息修改&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReturnBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>400</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回上一层</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyBtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="BGRLabel">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-20</y>
     <width>781</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/backimage.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CdanLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic;&quot;&gt;请输入应聘者编号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CdanLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>59</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>701</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TableBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Rider</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HR_View</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AuditResults</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ArrivalResults</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic;&quot;&gt;姓名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>109</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消修改</string>
   </property>
  </widget>
  <zorder>BGRLabel</zorder>
  <zorder>modifyBtn</zorder>
  <zorder>SearchBtn</zorder>
  <zorder>TitleLabel</zorder>
  <zorder>ReturnBtn</zorder>
  <zorder>CdanLabel</zorder>
  <zorder>CdanLineEdit</zorder>
  <zorder>tableView</zorder>
  <zorder>TableBox</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>BackButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
